src/main/views/courts/courts.njk

```diff
@@ -44,10 +41,37 @@
 
     <div id='numberOfCourts' class='govuk-hint'></div>
 
+    {% set courtItems = [] %}
+    {% for court in courts|sort(false, true, 'name') %}
+      {% set courtItems = (courtItems.push([
+        {
+          'attributes': {
+            'type': 'name',
+            'value': court.name,
+            'name': court.slug
+          },
+          'text': court.name
+        },
+        {
+          'attributes': {'type': 'displayed', 'value': court.displayed},
```

Comment:
I don't think the property names need to be in quotes e.g. I think you can just use: attributes: {.....}